package.json

```diff
@@ -88,10 +88,22 @@
     "yarn": ">= 1.0.0"
   },
   "lint-staged": {
-    "*.js": ["eslint --fix", "git add"],
-    "*.json": ["prettier --write", "git add"],
-    "*.less": ["prettier --write", "git add"],
-    "*.md": ["prettier --write", "git add"]
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.json": [
+      "prettier --write",
+      "git add"
+    ],
+    "*.less": [
+      "prettier --write",
+      "git add"
+    ],
+    "*.md": [
+      "prettier --write",
+      "git add"
+    ]
```

Comment:
@dependabot maybe we could define somewhere what to run after updating the dependencies.. in our case: `yarn fix:config`

Reply:
Sorry, I missed this one (looks like Dependabot doesn't pick up code comments, so I'll fix that now).

I'd rather we didn't mess with the file format! https://github.com/dependabot/dependabot-core/pull/146 should fix this.

Reply:
Both the above issues (Dependabot not listening for review comments and Dependabot updating anything other than the version in your `package.json`) should now be fixed. 🎉